tests/unit/lms/views/lti/deep_linking_test.py

```diff
@@ -1,36 +1,54 @@
 from unittest import mock
+from unittest.mock import create_autospec, sentinel
 
 import pytest
+from h_matchers import Any
 
+from lms.models import ApplicationInstance
 from lms.resources import LTILaunchResource
 from lms.resources._js_config import JSConfig
-from lms.views.lti.deep_linking import deep_linking_launch
+from lms.views.lti.deep_linking import DeepLinkingFieldsViews, deep_linking_launch
 
-pytestmark = pytest.mark.usefixtures(
-    "application_instance_service", "course_service", "lti_h_service"
-)
 
+@pytest.mark.usefixtures("application_instance_service", "lti_h_service")
+class TestDeepLinkingLaunch:
+    def test_it(
+        self, context, pyramid_request, lti_h_service, application_instance_service
+    ):
+        application_instance_service.get_current.return_value = create_autospec(
```

Comment:
The application_instance_service is wired up to return a real `application_instance` on `get_current`. 

This makes sense in a lot of tests but needed this here to be able to make assertions on how we could the result.